2023-07-21 10:19:17,412 INFO    MainThread:28165 [internal.py:wandb_internal():90] W&B internal server running at pid: 28165, started at: 2023-07-21 10:19:17.412196
2023-07-21 10:19:17,414 DEBUG   HandlerThread:28165 [handler.py:handle_request():141] handle_request: run_start
2023-07-21 10:19:17,414 INFO    WriterThread:28165 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230721_101917-2he72wc7/run-2he72wc7.wandb
2023-07-21 10:19:17,415 DEBUG   HandlerThread:28165 [meta.py:__init__():35] meta init
2023-07-21 10:19:17,415 DEBUG   HandlerThread:28165 [meta.py:__init__():49] meta init done
2023-07-21 10:19:17,416 DEBUG   HandlerThread:28165 [meta.py:probe():209] probe
2023-07-21 10:19:17,417 DEBUG   HandlerThread:28165 [meta.py:_setup_git():199] setup git
2023-07-21 10:19:17,424 DEBUG   HandlerThread:28165 [meta.py:_setup_git():206] setup git done
2023-07-21 10:19:17,424 DEBUG   HandlerThread:28165 [meta.py:_save_pip():53] save pip
2023-07-21 10:19:17,424 DEBUG   HandlerThread:28165 [meta.py:_save_pip():67] save pip done
2023-07-21 10:19:17,424 DEBUG   HandlerThread:28165 [meta.py:probe():247] probe done
2023-07-21 10:19:21,311 WARNING MainThread:28165 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-07-21 10:19:21,451 WARNING MainThread:28165 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-07-21 10:19:21,451 ERROR   MainThread:28165 [internal.py:wandb_internal():156] Internal process interrupted.
2023-07-21 10:19:22,313 INFO    WriterThread:28165 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230721_101917-2he72wc7/run-2he72wc7.wandb
2023-07-21 10:19:22,314 INFO    HandlerThread:28165 [handler.py:finish():806] shutting down handler
2023-07-21 10:19:22,417 INFO    SenderThread:28165 [sender.py:finish():1111] shutting down sender
2023-07-21 10:19:22,418 INFO    MainThread:28165 [internal.py:handle_exit():80] Internal process exited
